content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -550,34 +398,40 @@ $ curl $INGRESS_HOST/headers -s -o /dev/null -w "%{http_code}\n"
 200
 {{< /text >}}
 
-Now, add a policy that requires end-user JWT for `httpbin.foo`. The next command assumes there is no service-specific policy for `httpbin.foo` (which should
-be the case if you run [cleanup](#cleanup-part-2) as described). You can run `kubectl get policies.authentication.istio.io -n foo` to confirm.
+Now, add a request authentication policy that requires end-user JWT for the ingress gateway.
 
 {{< text bash >}}
-$ cat <<EOF | kubectl apply -n foo -f -
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
+$ kubectl apply -f - <<EOF
+apiVersion: "security.istio.io/v1beta1"
+kind: "RequestAuthentication"
 metadata:
   name: "jwt-example"
+  namespace: istio-system
 spec:
-  targets:
-  - name: httpbin
-  origins:
-  - jwt:
-      issuer: "testing@secure.istio.io"
-      jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
-  principalBinding: USE_ORIGIN
+  selector:
+    matchLabels:
+      istio: ingressgateway
+  jwtRules:
+  - issuer: "testing@secure.istio.io"
+    jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
 EOF
 {{< /text >}}
 
-The same `curl` command from before will return with 401 error code, as a result of server is expecting JWT but none was provided:
+Submit the policy in the same namespace as the workload it selects, `ingressgateway` in this case. The namespace you need to specify is then `istio-system`.
```

Comment:
Submit policy seems a non conventional usage, compared to the rest of the wording.

Reply:
I agree, WDYT about:
```suggestion
Apply the policy to the namespace of the workload it selects, `ingressgateway` in this case. The namespace you need to specify is then `istio-system`.
```